planet/api/experimental/basemaps_client.py

```diff
@@ -71,3 +81,23 @@ def list_quads_in_mosaic(
             for s in quad_list:
                 yield s
             url = resp.json().get('_links', {}).get('_next')
+
+    def get_quads_in_mosaic_for_region(self, mosaic_id, aoi):
+        url = self.base_url + f'basemaps/v1/mosaics/{mosaic_id}/quads/search?minimal=true'
```

Comment:
Uh oh, we can't use f strings here because some people/systems installing this library might still be using python 2 😬 (check out the failures in your pull request's CI test run)

Deprecating python 2 support is worth considering but outside of the scope of your current pull request.

Reply:
yeaaaah, you know I have Opinions here @strixcuriosus ;) TL;DR: deprecating Python 2 is on longer-term roadmap but for now we do still support it.

@jmanyika there's a `tox.ini` config in this repo that allows you to use a Python testing tool called [tox](https://tox.readthedocs.io/en/latest/); this makes it easier to run the tests against multiple Python versions locally. If you're interested in trying that, check out [these notes](https://github.com/planetlabs/planet-client-python/wiki#managing-python-2--3).